twiml/voice/pay/prompt/requireMatchingInputs/basic/prompt-requireMatchingInputs-basic.7.x.py

```diff
@@ -0,0 +1,14 @@
+from twilio.twiml.voice_response import Pay, Prompt, VoiceResponse, Say
+
+response = VoiceResponse()
+pay = Pay(payment_method='ach-debit', charge_amount='13.22')
+prompt = Prompt(for_='bank-account-number')
+prompt.say(
```

Comment:
nit: this can be on one line.

Reply:
Oh, hmm I wonder why it printed like that! Something in the twiml generator, perhaps?